mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -362,19 +362,20 @@ float ControllerAction::calculateGlobalPathLengthLeft(
   const mbf_msgs::ExePathGoal &goal = *(goal_handle.getGoal().get());
   const std::vector<geometry_msgs::PoseStamped> &plan = goal.path.poses;
 
-  unsigned int clostest_point_index = 0;
+  int closest_point_index = 0;
```

Comment:
Instead of changing data types, we should special case plans of size 0 and size 1.  For size 0, the we can return 0 distance or something.  For size 1, we just do euclidean distance.  For size >= 2, then we can do this logic.